src/components/placeorder/PlaceOrder.js

```diff
@@ -24,6 +25,11 @@ const PlaceOrder = () => {
 			<div className={Layout.pricecard}>
 				<PriceCard />
 			</div>
+			<div>
+				<CollapsibleDiv name= {"Payment Options"}>
```

Comment:
I need a props here collapsed if it is true the div will be collapsed other wise it will be open by default